docs/history.md

```diff
@@ -1,5 +1,17 @@
 # Version History
 
+## [1.0.0-alpha3](https://www.github.com/bcoe/functions-framework-dotnet-test/compare/v1.0.0-alpha2...v1.0.0-alpha3) (2020-09-11)
```

Comment:
I'd expect the link to be to a tag rather than a diff. Not a big issue, just an expectation.